mkdocs.yml

```diff
@@ -57,6 +57,7 @@ nav:
       - UI Coding Guidelines: development-guidelines/UICodingGuidelines.md
       - UI Styling Guidelines: development-guidelines/UIStylingGuidelines.md
       - Interacting with FileSystem: development-guidelines/UsingIFileSystem.md
+      - Using Workspaces: development-guidelines/UsingWorkspaces.md
```

Comment:
Might be a good idea to move this 1 line up, to put it alongside the UI stuff, or move all the UI stuff to the bottom to create a grouping.